cmake/external_protobuf.cmake

```diff
@@ -21,14 +21,15 @@ include(ExternalProject)
 # Download and install Google Protobuf ...
 #------------------------------------------------------------------------------
 
-set(PROTOBUF_GIT_REPO_URL https://github.com/google/protobuf.git)
-set(PROTOBUF_GIT_BRANCH origin/3.5.x)
+# This version of PROTOBUF is required by Microsoft ONNX Runtime.
+set(NGRAPH_PROTOBUF_GIT_REPO_URL "https://github.com/google/protobuf.git")
+set(NGRAPH_PROTOBUF_GIT_BRANCH "origin/3.5.x")
```

Comment:
Can we use a tag instead of a branch. "v3.5.2" is the latest tag on the branch "origin/3.5.x"